<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Euclid::PhzQtUI::DialogAddGalEbv</class>
 <widget class="QDialog" name="Euclid::PhzQtUI::DialogAddGalEbv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Galactic E(B-V) to the catalog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Look-up Galactic E(B-V) in Planck map and add it as a &quot;GAL_EBV&quot; column to the input catalog </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select the name of the new input catalog file:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txt_name"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_process">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_create">
       <property name="text">
        <string>Add the E(B-V)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
